cli/azd/pkg/config/manager.go

```diff
@@ -95,3 +97,28 @@ func GetUserConfigFilePath() (string, error) {
 
 	return filepath.Join(configPath, "config.json"), nil
 }
+
+// GetUserConfig gets the Config for the current user, given a manager. If the config file does not exist, an empty
+// configuration object is returned.
+func GetUserConfig(configManager Manager) (Config, error) {
```

Comment:
This makes sense to move here since other consumers of config will need this.

Reply:
I found myself having to open-code the saving logic, I think I will just create a SaveUserConfig as well?

Reply:
I ended up adding a SaveUserConfig as well.